docs/6.x/changelog.md

```diff
@@ -299,7 +304,19 @@ LTS starts with `3.51.0` with minor backwards compatible changes added over time
     This release fixes a security vulnerability in teleport. Please see
     [Teleport Announcement](https://github.com/gravitational/teleport/releases/tag/v4.0.5) for more information.
 
-### 5.5.20 (September 18th, 2019)
+### 5.5.21 LTS (September 26th, 2019)
+
+#### Improvements
+
```

Comment:
* Update github.com/gravitational/monitoring-app to 5.5.3 ([#644](https://github.com/gravitational/gravity/pull/644)).
  * Disable NodePort access to Kapacitor service.
  * Randomly generate passwords for superuser, telegraf, heapster and grafana users in InfluxDB database during installation and upgrades.